codes/quantum/qubits/subsystem/subsystem-gen-shor.yml

```diff
@@ -0,0 +1,46 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+code_id: subsystem-gen-shor
+physical: qubits
+logical: qubits
+
+
+name: 'Subsystem-Generalized Shor Code'
+introduced: '\cite{arXiv:quant-ph/0610088v2}'
+
+
+description: |
+  For every generalized Shor code, there is a subsystem code which has the same parameters as the subspace version, but requires significantly fewer stabilizer measurements, resulting in a much simpler error recovery routine.
```

Comment:
Is generalized Shor code in the zoo? It would be prudent to add it if not as then this entry can lean quite heavily on that entry.

Reply:
Yes, that code is in quantum_parity.yml. Would be great if the detail re construction relating to the subspace version could be merged into that entry, so that any subsystem details would make the meat of this one.